apps/server/src/apps/tldraw.app.ts

```diff
@@ -0,0 +1,29 @@
+/* istanbul ignore file */
+/* eslint-disable no-console */
+import { NestFactory } from '@nestjs/core';
+import { install as sourceMapInstall } from 'source-map-support';
+import { TldrawModule } from '@src/modules/tldraw';
+import { AppStartLoggable } from '@src/apps/helpers/app-start-loggable';
+import { Logger } from '@src/core/logger';
+import * as WebSocket from 'ws';
+import { WsAdapter } from '@nestjs/platform-ws';
+
+async function bootstrap() {
+	sourceMapInstall();
+	const nestApp = await NestFactory.create(TldrawModule);
+	const wss = new WebSocket.Server({ noServer: true });
+	const logger = await nestApp.resolve(Logger);
+	nestApp.useWebSocketAdapter(new WsAdapter(wss));
+	nestApp.enableCors();
+	const port = 3344;
+	await nestApp.listen(port);
```

Comment:
I think you can just use `nestApp.init()` if you don't need to expose additional HTTP server (which you don't right now as far as I know as you don't have any HTTP controllers defined).